README.rst

```diff
@@ -104,7 +104,23 @@ Once chromedriver is installed, the tests can be run using pytest:
 
 .. code:: bash
 
+    export PYTHONPATH=$(pwd)
```

Comment:
i'm not sure why this is showing up. while in my branch, `git diff master..` does not show this change.

Reply:
Yea not sure either -- if it messes anything up should be easy to clean up after

Reply:
@cauthu Your base branch and local master probably don’t match upstream master.

Reply:
i'll just merge and if it breaks i'll fix :)

Reply:
@cbguder so i guess it merged properly, i.e., the readme.md change was not in the diff /shrug